code/datums/quirks/negative_quirks/addict.dm

```diff
@@ -17,17 +11,19 @@
 	var/obj/item/accessory_type //! If this is null, an accessory won't be spawned.
 	var/process_interval = 30 SECONDS //! how frequently the quirk processes
 	var/next_process = 0 //! ticker for processing
-	var/drug_flavour_text = "Better hope you don't run out..."
+	var/drug_flavour_text = "Better hope you don't run out... of what, exactly? You don't know."
+	abstract_parent_type = /datum/quirk/item_quirk/addict
 
-/datum/quirk/item_quirk/junkie/add_unique(client/client_source)
+/datum/quirk/item_quirk/addict/add_unique(client/client_source)
 	var/mob/living/carbon/human/human_holder = quirk_holder
+	var/addiction
```

Comment:
What is this declaration for?

Reply:
I was trying to fix the TGUI i think, removing it